### PR TITLE
Update to iroha:2.0.0-pre-rc.22.2

### PR DESCRIPTION
**Pull request by arndey**

### CODE REVIEW

File: modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Extensions.kt [hunk 1]
```diff
@@ -1,61 +1,17 @@
...
-import jp.co.soramitsu.iroha2.generated.SignedTransactionV1
-import jp.co.soramitsu.iroha2.generated.SocketAddr
-import jp.co.soramitsu.iroha2.generated.SocketAddrHost
-import jp.co.soramitsu.iroha2.generated.StringWithJson
-import jp.co.soramitsu.iroha2.generated.TransactionPayload
-import jp.co.soramitsu.iroha2.generated.TriggerId
-import jp.co.soramitsu.iroha2.generated.TriggerOfTriggeringFilterBox
-import jp.co.soramitsu.iroha2.generated.TriggeringFilterBox
-import jp.co.soramitsu.iroha2.generated.Value
+import jp.co.soramitsu.iroha2.generated.* // ktlint-disable no-wildcard-imports
```
Comment (Mingela) [inline modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Extensions.kt:8]: I doubt we need all, why not go back to explicit imports? any convention to limit import lines?

File: modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/client/blockstream/BlockStreamSubscription.kt [hunk 2]
```diff
@@ -105,45 +106,42 @@ open class BlockStreamSubscription private constructor(
...
+            method = HttpMethod.Get,
         ) {
             try {
                 logger.debug("WebSocket opened")
                 send(BlockSubscriptionRequest.encode(request).toFrame())
                 val idsToRemove = mutableListOf<UUID>()
 
                 for (frame in incoming) {
-                    logger.debug("Received frame: {}", frame)
+                    logger.info("Received frame: {}", frame)
```
Comment (Mingela) [inline modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/client/blockstream/BlockStreamSubscription.kt]: debug seems more sensible; for the entire file

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/PeerTest.kt [hunk 3]
```diff
@@ -49,16 +55,52 @@ class PeerTest : IrohaTest<AdminIroha2Client>() {
...
+        val p2pPort = DEFAULT_P2P_PORT + PEER_AMOUNT
         val alias = "iroha$p2pPort"
         val address = "$alias:$p2pPort"
         val keyPair = generateKeyPair()
         val payload = keyPair.public.bytes()
 
-        startNewContainer(keyPair, alias).use {
-            registerPeer(address, payload)
-            assertTrue(isPeerAvailable(address, payload))
+//        val kp = keyPairFromHex(
```
Comment (Mingela) [inline modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/PeerTest.kt]: commented code?

File: modules/client/src/test/resources/create_nft_for_alice_smartcontract/Cargo.toml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "create_nft_for_alice_smartcontract"
-version = "2.0.0-pre-rc.3"
+version = "2.0.0-pre-rc.22.1"
```
Comment (Mingela) [inline modules/client/src/test/resources/create_nft_for_alice_smartcontract/Cargo.toml]: should be updated to 22.2?

Comment (arndey) [reply modules/client/src/test/resources/create_nft_for_alice_smartcontract/Cargo.toml]: > should be updated to 22.2?

IDK, needs to ask Iroha team

Comment (mversic) [reply modules/client/src/test/resources/create_nft_for_alice_smartcontract/Cargo.toml]: yes, update it to rc22.2

File: modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Extensions.kt (hunk 1 above)
Comment (arndey) [reply modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Extensions.kt:8]: too many imports from `generated.*`

**Review by Mingela — APPROVED** [review]

**Review by gv-timur — APPROVED** [review]

**Review by mversic — APPROVED** [review]

**Pull request merged**